integration/benchmark/server/systemd/flow-tps.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Flow TPS tests - generate list of merge commit hashes and run TPS tests against each one
+
+[Service]
+Type=oneshot
+ExecStart=/opt/flow-go/integration/benchmark/server/hello.sh
+ExecStart=/opt/flow-go/integration/benchmark/server/control.sh
+ExecStart=/opt/flow-go/integration/benchmark/server/bench.sh
```

Comment:
nitpick: Rather than depending on `ExecStart` ordering could we condense these into a single bash script that is responsible for orchestrating the execution of the other scripts? This would allow us to maintain distinct scripts while ensuring the ordering of execution is not placed in `systemd`

Reply:
I would prefer to leave the 3 scripts independent of each other and orchestrated by `systemd`. I tested `systemd` by easily adding / removing scripts from this list without the scripts having to know about each other. I don't see an issue with `systemd` orchestrating the scripts.